### PR TITLE
Move grpc to common

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Move the grpcV2 implementation to common-sdk and create a f.unction in the web-sdk to create a client that uses grpc-web.
Run grpcV2 tests both with the regular grpc and grpc-web

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/nodejs/test/clientV2.test.ts [hunk 1]
```diff
@@ -232,44 +297,50 @@ test('Failed invoke contract', async () => {
...
+);
+
+test.each([clientV2, clientWeb])(
+    'Invoke contract same in v1 and v2 on v1 contract',
+    async (client) => {
+        const context = {
             invoker: testAccount,
             contract: {
-                index: 6n,
+                index: 81n,
```
Comment (rasmus-kirk) [inline packages/nodejs/test/clientV2.test.ts:328]: I'm sure it's fine, but why change the contract?

**Review by rasmus-kirk — COMMENTED** [review]
Generally LGTM

File: packages/nodejs/test/clientV2.test.ts (hunk 1 above)
Comment (shjortConcordium) [reply packages/nodejs/test/clientV2.test.ts:328]: It is another test, the git diff is just weird. (Above I still have the test with the index 6 contract)
This one invokes a v1 contract, which the one on index 81 is, and the 6 one is a v0 contract.

Comment (rasmus-kirk) [reply packages/nodejs/test/clientV2.test.ts:328]: Makes sense

**Comment by rasmus-kirk** [comment]
When you make the CI happy I'll approve it

**Review by rasmus-kirk — APPROVED** [review]

**Pull request merged**